src/client/app/audit/audit.component.ts

```diff
@@ -7,19 +7,20 @@ import { Observable } from 'rxjs/Observable';
 import * as _ from 'lodash';
 import * as moment from 'moment';
 
+import { UserService } from 'app/admin/user';
+import {
+	PagingOptions, TableSortOptions,
+	SortDisplayOption, SortDirection
+} from 'app/shared';
 
 import { AuditService } from './audit.service';
 import { AuditViewDetailModalContext, AuditViewChangeModal, AuditViewDetailModal } from './audit-view-change.component';
-import { PagingOptions } from '../shared/pager.component';
-import { TableSortOptions } from '../shared/pageable-table/pageable-table.component';
-import { SortDisplayOption, SortDirection } from '../shared/result-utils.class';
-import { UserService } from '../admin/users.service';
 import { AuditOption } from './audit.classes';
 
 
 @Component({
 	selector: 'audit',
-	templateUrl: './audit-list.component.html'
+	templateUrl: './audit-list.component.html',
```

Comment:
I don't have a problem with discussing if we want to start using this, but it should be a discussion. In the interim I'd rather avoid having mixed use. In this case can we revert to not using it (particularly since this change is not directly related to the purpose of this branch/PR?